docusaurus/docusaurus.config.js

```diff
@@ -27,6 +27,17 @@ const config = {
   organizationName: "airbytehq", // Usually your GitHub org/user name.
   projectName: "airbyte", // Usually your repo name.
 
+  // Adds one off script tags to the head of each page
+  // .e.g <script async data-api-key="..." id="unifytag" src="..."></script>
+  scripts: [
+    {
+      src: "https://cdn.unifygtm.com/tag/v1/unify-tag-script.js",
+      async: true,
+      id: "unifytag",
+      "data-api-key": "wk_BEtrdAz2_2qgdexg5KRa6YWLWVwDdieFC7CAHkDKz",
+    }
+  ],
```

Comment:
looks reasonable according to [this](https://docusaurus.io/docs/api/docusaurus-config#scripts)!

A few follow ups: 
- why are some keys not in quotes while some are? 
- have we built the docs locally to validate that we get the output we want?
- do we need something for `type="module"`?

Reply:
in no particular order

- I did build and validate the script was in the head of the page
- The one key being in quotes is a js thing. Linter wants no quotes with the exception of situations where they are unavoidable. Which is here given the kebab case
- I missed type module!!